apps/zotonic_mod_authentication/priv/templates/_dialog_session_close.tpl

```diff
@@ -0,0 +1,14 @@
+
+<p>
+	{_ Every sessions will be interrupted and logged out of your account. _}<br>
```

Comment:
Maybe: 

`Every session will be interrupted and you will be logged out.`

Reply:
Sounds good, done in c52ab1f2abfec75e16bcc7895ed8abcab04d010e